src/SignalR/server/StackExchangeRedis/src/Internal/RedisProtocol.cs

```diff
@@ -104,101 +108,118 @@ public byte[] WriteAck(int messageId)
             // * An 'int': The Id of the command being acknowledged.
             // Any additional items are discarded.
 
-            var writer = MemoryBufferWriter.Get();
+            var memoryBufferWriter = MemoryBufferWriter.Get();
             try
             {
-                MessagePackBinary.WriteArrayHeader(writer, 1);
-                MessagePackBinary.WriteInt32(writer, messageId);
+                var writer = new MessagePackWriter(memoryBufferWriter);
+
+                writer.WriteArrayHeader(1);
+                writer.WriteInt32(messageId);
```

Comment:
Use `writer.Write(int)` here instead of `writer.WriteInt32` to preserve old (and more compact) msgpack format.

Reply:
@BrennanConroy 
same here ? still using `WriteInt32` ?

Reply:
done